python/lsst/ts/dateloc/date_profile.py

```diff
@@ -38,51 +55,98 @@ def __call__(self, timestamp):
         Returns
         -------
         (float, float)
-            A tuple of the Modified Julian Date and Local Sidereal Time (radians).
+            A tuple of the Modified Julian Date and Local Sidereal Time
+            (radians).
         """
         self.update(timestamp)
         return (self.mjd, self.lst_rad)
 
     def __get_timestamp(self, dt):
-        """float: Return a timestamp from the datetime instance.
+        """Get timestamp
+
+        Parameters
+        ----------
+        dt : datetime
+            Date, in `datetime` format, to convert to timestamp.
```

Comment:
Isn't the convention to use the back ticks on `datetime` in the line above this? So like this:

```
dt : `datetime`
    Date to convert to timestamp.
```

I think this could (should?) be changed in all other cases as well.